xbmc/video/dialogs/GUIDialogVideoBookmarks.cpp

```diff
@@ -339,8 +342,13 @@ bool CGUIDialogVideoBookmarks::AddBookmark(CVideoInfoTag* tag)
     {
       g_renderManager.Capture(thumbnail, width, height, CAPTUREFLAG_IMMEDIATELY);
 
+#if !defined(HAS_LIBAMCODEC)
       if (thumbnail->GetUserState() == CAPTURESTATE_DONE)
       {
+#else//HAS_LIBAMCODEC
+      {
+        CScreenshotAML::CaptureVideoFrame(thumbnail->GetPixels(), width, height, false);
```

Comment:
copying into the pixel buffer RenderCapture is rather hacky. @Memphiz needs some rework after https://github.com/xbmc/xbmc/pull/8613